### PR TITLE
CDAP-7175 Add documentation for upgrade tool with authorization enabled

### PR DESCRIPTION
**Pull request by rohitsinha54**
Docs Quick Build: https://builds.cask.co/browse/CDAP-DQB129-1

### CODE REVIEW

File: cdap-docs/admin-manual/source/upgrading/cloudera.rst [hunk 1]
```diff
@@ -44,6 +44,10 @@ CDAP:
 
 #. Stop all flows, services, and other programs in all your applications.
 
+#. If you are upgrading *authorization* enabled CDAP instance you will need to give *cdap* user *ADMIN* privilege on
+   all existing CDAP Namespaces. Note: *ADMIN* privilege does not give *cdap* user any access (*READ/WRITE*)
```
Comment (bdmogal) [inline cdap-docs/admin-manual/source/upgrading/cloudera.rst]: ... does not give READ/WRITE privileges. 

(I think "any access" adds to confusion)

File: cdap-docs/admin-manual/source/upgrading/cloudera.rst [hunk 2]
```diff
@@ -44,6 +44,10 @@ CDAP:
...
+   privileges on the Namespaces.
```
Comment (bdmogal) [inline cdap-docs/admin-manual/source/upgrading/cloudera.rst]: You may also want to mention that this is a temporary grant required only for the purpose of running the upgrade tool. Once upgrade completes, these grants can be revoked.

File: cdap-docs/admin-manual/source/upgrading/packages.rst [hunk 3]
```diff
@@ -41,6 +41,10 @@ and then restart CDAP:
 
 1. Stop all flows, services, and other programs in all your applications.
 
+#. If you are upgrading *authorization* enabled CDAP instance you will need to give *cdap* user *ADMIN* privilege on
+   all existing CDAP Namespaces. Note: *ADMIN* privilege does not give *cdap* user any access (*READ/WRITE*)
```
Comment (bdmogal) [inline cdap-docs/admin-manual/source/upgrading/packages.rst]: same comments here too

File: cdap-docs/admin-manual/source/upgrading/cloudera.rst [hunk 4]
```diff
@@ -44,6 +44,10 @@ CDAP:
 
 #. Stop all flows, services, and other programs in all your applications.
 
+#. If you are upgrading *authorization* enabled CDAP instance you will need to give *cdap* user *ADMIN* privilege on
```
Comment (bdmogal) [inline cdap-docs/admin-manual/source/upgrading/cloudera.rst]: If you are upgrading a CDAP instance with authorization enabled...

File: cdap-docs/admin-manual/source/upgrading/cloudera.rst [hunk 5]
```diff
@@ -44,6 +44,10 @@ CDAP:
...
+   all existing CDAP Namespaces. Note: *ADMIN* privilege does not give *cdap* user any access (*READ/WRITE*)
+   privileges on the Namespaces.
+
```
Comment (bdmogal) [inline cdap-docs/admin-manual/source/upgrading/cloudera.rst:54]: You may also want to mention that 

Alternatively, you can also disable authorization for purpose of upgrading CDAP, and then enable it again.

Good to also point to the authorization section in the admin manual for steps on how to enable/disable

**Comment by awholegunch** [comment]
@rohitsinha54 @bdmogal I've edited the paragraph in question; please review. LGTM.

Started a new Quick Build: https://builds.cask.co/browse/CDAP-DQB129-2

File: cdap-docs/admin-manual/source/upgrading/cloudera.rst [hunk 6]
```diff
@@ -44,6 +44,12 @@ CDAP:
 
 #. Stop all flows, services, and other programs in all your applications.
 
+#. If you are upgrading an *authorization-enabled* CDAP instance, you will need to give 
+   the *cdap* user *ADMIN* privileges on all existing CDAP namespaces. **Note:** the *ADMIN*
+   privilege does not give the *cdap* user *READ* or *WRITE* privileges on the namespaces.
+   As these are temporary grants required only for the purpose of running the upgrade tool, 
+   once the upgrade is complete, these grants can be revoked.
+
```
Comment (bdmogal) [inline cdap-docs/admin-manual/source/upgrading/cloudera.rst:54]: "Alternatively, you can also disable authorization before running the upgrade tool, and enable it again after the upgrade is completed."?

Also, perhaps a link to the authorization admin manual?

**Comment by awholegunch** [comment]
Now running a **Quick Build:** https://builds.cask.co/browse/CDAP-DQB129-3

**Comment by bdmogal** [comment]
lgtm

**Pull request merged**